tests/unit/stats/test_aggregation_data.py

```diff
@@ -205,18 +272,17 @@ def test_constructor_with_exemplar(self):
             counts_per_bucket=counts_per_bucket,
             bounds=bounds)
 
-        self.assertEqual(1, dist_agg_data.mean_data)
-        self.assertEqual(0, dist_agg_data.count_data)
-        self.assertEqual(0, dist_agg_data.min)
-        self.assertEqual(1, dist_agg_data.max)
-        self.assertEqual(sum_of_sqd_deviations,
-                         dist_agg_data.sum_of_sqd_deviations)
-        self.assertEqual([1, 1, 1], dist_agg_data.counts_per_bucket)
-        self.assertEqual([exemplar_1, exemplar_2], dist_agg_data.exemplars[2])
```

Comment:
This was asserting the wrong behavior: attaching all exemplars to the last bucket.